src/System.Private.CoreLib/src/System/Char8.cs

```diff
@@ -0,0 +1,69 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System
+{
+    /// <summary>
+    /// Represents a UTF-8 code unit, the elemental type of <see cref="Utf8String"/>.
+    /// </summary>
+    public readonly struct Char8 : IComparable<Char8>, IEquatable<Char8>
+    {
+        private readonly byte Value;
+
+        private Char8(byte value)
+        {
+            Value = value;
+        }
+
+        public static bool operator ==(Char8 a, Char8 b) => a.Value == b.Value;
```

Comment:
These should be named `left` and `right`, not `a` and `b`, according to the Framework Design Guidelines:

https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/naming-parameters

> **✓ DO** use `left` and `right` for binary operator overload parameter names if there is no meaning to the parameters.